docs/source/using/troubleshooting.adoc

```diff
@@ -113,6 +113,15 @@ cause passwords to fail.
 Workaround: When creating and entering passwords, wrap the string with
 single quotes in the following format: `<password>`
 
+NOTE: The characters `", $, ` and \` are still interpreted by the shell,
```

Comment:
> Have you tried passing a password with a space? 'hello world' for example? The proper solution is to put the password in single quotes when calling he ssh command.

This is something we can't set as per access.redhat.com portal if you try to put space or any below character then you can't set your RHN password. But I understand your intention here, also http://teaching.idallen.com/cst8207/14w/notes/445_quotes_for_remote.html#quoting-variables-in-remote-command-lines helped me to understand the ssh behavior.

```
<extract from access.redhat.com>
Please correct the following errors:

Password must be ASCII and cannot contain the following special characters (") (<) (>) (space)
```